ui/src/components/Publication/RelatedPublications/Card/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import * as SolidIcons from '@heroicons/react/24/solid';
+import * as Interfaces from '@/interfaces';
+import * as Components from '@/components';
+import * as Helpers from '@/helpers';
+import * as Config from '@/config';
+
+type Props = {
+    crosslink: Interfaces.Crosslink;
+};
+
+const Card: React.FC<Props> = (props): React.ReactElement => {
+    const version = props.crosslink.linkedPublication.latestLiveVersion;
+    return (
+        <div className="rounded-md shadow my-2">
+            <div
+                className={`rounded-t-lg border-b border-teal-500 bg-white-50 p-4 transition-colors duration-500 dark:bg-grey-700`}
+            >
+                <p className="mb-4 block min-h-[3rem] font-montserrat text-sm font-bold leading-snug text-grey-900 transition-colors duration-500 dark:text-white-50 lg:min-h-[4rem] 2xl:text-md">
+                    {version.title.length > 80 ? Helpers.truncateString(version.title, 80) : version.title}
+                </p>
+                <div className="flex">
+                    <span className="w-1/2 text-xs font-medium tracking-wide text-grey-800 transition-colors duration-500 dark:text-grey-100 pr-2">
+                        By{' '}
+                        {
+                            // TODO: This should use the abbreviate user name helper function which is not in main yet
```

Comment:
Made a note to come back to this?

Reply:
Yes, and now it's in main, so I've used it :) just pushed.